tests/providers/common/io/xcom/test_backend.py

```diff
@@ -181,7 +182,7 @@ def test_value_storage(self, task_instance, session):
             run_id=task_instance.run_id,
             session=session,
         )
-        assert self.path in qry.first().value
+        assert self.path_normalized in qry.first().value
```

Comment:
We’ll need better tests for this behaviour

Reply:
Addressed via checking for the exact serialized path